packages/nvidia_nat_langchain/src/nat/plugins/langchain/chat_models/huggingface.py

```
@@ -0,0 +1,168 @@
+                logger.debug("Chat template application failed: %s, using fallback", e)
+                last_msg = messages[-1]
+                text = last_msg.get("content", str(last_msg)) if isinstance(last_msg, dict) else str(last_msg)
+        else:
+            text = str(messages)
+        return text
+
+    def _generate(
+        self,
+        messages: list[BaseMessage],
+        stop: list[str] | None = None,
+        run_manager: CallbackManagerForLLMRun | None = None,
```

Ruff (0.14.10)

Unused method argument: `run_manager`

(ARG002)